mne/preprocessing/maxwell.py

```diff
@@ -102,13 +112,37 @@ def maxwell_filter(raw, origin=(0, 0, 40), int_order=8, ext_order=3,
     pS_tot = pinv(S_tot, cond=1e-15)
     # Compute multipolar moments of (magnetometer scaled) data (Eq. 37)
     mm = np.dot(pS_tot, data * coil_scale[:, np.newaxis])
+
     # Reconstruct data from internal space (Eq. 38)
-    recon = np.dot(S_in, mm[:S_in.shape[1], :])
+    recon = np.dot(S_in, mm[:S_in.shape[1], :]) / coil_scale[:, np.newaxis]
+
+    if st_dur is not None:
+        # Reconstruct data from external space and compute residual
+        recon_out = np.dot(S_out, mm[S_in.shape[1]:, :]) /\
+            coil_scale[:, np.newaxis]
+        resid = data - (recon + recon_out)
+
+        # Generate time points to break up data in to windows
+        t_starts = raw.time_as_index(np.arange(times[0], times[-1], st_dur))
+        t_ends = np.arange(times[0] + st_dur, times[-1], st_dur)
+        # TODO: Decide how to handle last window if it's shorter length
```

Comment:
@staulu, how is the last window of the tSSS processing handled if the buffer length doesn't match the last window (e.g., the final window of a 25 sec dataset with a 10 second buffer length). In some of your earlier scripts, it looks like the final window is skipped.

Also, MaxFilter throws an error if <500 samples are in the specified buffer window. I'll probably add that -- is 500 samples a good value to keep using?